woocommerce/woocommerce
# Merge blocks branch: add 8 product blocks and related API endpoints

This PR contains code from #22809, #22831, #22832, #22856, #22858, #22941 – which adds the `wc-blocks/v1/*` endpoints, tests for the new endpoints, and all the block files. This branch will add 8 new blocks to the block editor, under the new section "WooCommerce":

![screen shot 2019-03-06 at 6 12 27 pm](https://user-images.githubusercontent.com/541093/53920534-862f3680-403b-11e9-96dd-d251e00aff44.png)

I've also bumped the `@woocommerce/block-library` package to v2.0.0-beta, as all existing work has wrapped up and we're only looking out for bugs now.

### How to test the changes in this Pull Request:

1. Add a new post or page
2. Click the + to add a block, or start typing a block name after a `/`
3. You should be able to add any block to your content
4. Add more blocks if you want
5. Publish your post
6. View it on the front end, it should look very close to the preview (themes may do slightly different things)
